src/isar/services/service_connections/mqtt/robot_status_publisher.py

```diff
@@ -86,8 +90,12 @@ def run(self) -> None:
         while True:
             try:
                 self.robot_status = self.robot.robot_status()
-            except RobotCommunicationException:
-                self.logger.warning(
-                    "Failed to get robot status due to a communication exception"
+            except (
+                RobotCommunicationException,
+                RobotAPIException,
+                RobotException,
```

Comment:
Yes, however I wanted to be specific in showing which exceptions are "expected" for this interface function. I understand that it may be considered bloat, but the interface functions have been documented with specific exceptions (these three) and I wanted to be clear that they are here. I also want to catch any other `RobotExceptions` which might be raised and in this case the behavior for all cases is the same.

Reply:
Fine by me